docs/docs/deploying.md

```diff
@@ -52,6 +52,11 @@ Set the `REDIS_URL` environment variable to a connection string for your Redis i
 
 Set the `PASSPHRASES` environment variable to some secret passphrase. You can also specify multiple by `:`-separating them.
 
+#### Optional
+
+Set the `REDIS_TLS_CA_FILE` environment variable to the path of your certificate if your Redis connection requires you to provide a self-signed certificate.
```

Comment:
I'm thinking, is there a way to pass in the certificate as a string to ioredis? Bind-mounting sounds like a hard thing to do, especially when deploying Quirrel to something like Heroku. Maybe we should allow passing in the CA file as a base64-encoded string or something

Reply:
From experience: in my application I had it as a base64 encoded string. At some point I had to realize, that some providers have max length for environment variables to be stored (personally experienced it with scaleway, [which has a limit of 1000 chars](https://www.scaleway.com/en/docs/compute/containers/reference-content/containers-limitations/)).

Reply:
alternative could be to provide both options, REDIS_TLS_CA_FILE & REDIS_TLS_CA

Reply:
will do this, even though its a adds a bit too much complexity for my taste

Reply:
then let's do both. there's definitely providers that don't allow files at all, so we should try to support both.

Reply:
well, I mean one can always build his own image with just a line COPY pem to place...

Reply:
Idea how to explain this here so someone still understands it?